runtime/requirements.txt

```diff
@@ -1,3 +1,3 @@
 protobuf==3.2.0
-pylint
--r ../hibike/requirements.txt
+pylint==1.7.2
+pyserial==3.2.1
```

Comment:
I'm scared that changing this from installing `hibike/requirements.txt` will cause problems in the future if Hibike adds new dependencies that need to be installed.